components/spinner/test/index.js

```diff
@@ -0,0 +1,20 @@
+/**
+ * External dependencies
+ */
+import { expect } from 'chai';
+import { shallow } from 'enzyme';
+
+/**
+ * Internal dependencies
+ */
+import Spinner from '../';
+
+describe( 'Spinner', () => {
+	describe( 'basic rendering', () => {
+		it( 'should match the following element', () => {
+			const element = <span className="spinner is-active" />;
+			const spinner = shallow( <Spinner /> );
+			expect( spinner.matchesElement( element ) ).to.be.true();
```

Comment:
Where do we put the limit between testing the markup or not? 

I think you're doing an awesome job testing all these components, but do we really need to add this test specifically? Does it avoid any regression, what's the value here since we're basically copy/pasting the content of the component here?